### DISCUSSION TITLE
Grove - Speaker | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#63] [Q&A]
# Grove - Speaker | Seeed Studio Wiki

The Grove- Speaker is a module which consists of power amplification and voice outputs. The loudness can be adjusted by the on-board potentiometer. With different input frequencies, the loud-speaker generates different tones. Coding the music into arduino, DIY your own music box!

https://wiki.seeedstudio.com/Grove-Speaker/

**Comment by naim514** [comment]
hi please i want to connect my potentiometer to this speaker to control sound volume by microbit but i didn´t succeed .help please
i connected gnd of potentiometer to gnd of microbit ,vcc to 5v and where i will connect out pin 

for speaker i connect gnd to gnd ,vcc to vcc and sig to p1 of microbit

help please thx in advance
regards

**Comment by MatthewJeffson** [comment]
Welcome to our product documentation platform! We value user feedback and contributions, so we couldn’t be more grateful that you can help us improving the documentation.

Hence, if you find obvious errors in the product documentation, or you have additional ideas and insights about using products, please [contribute](https://wiki.seeedstudio.com/Contribution-Guide) to us.

If you need technical support, you can email it to [fae@seeed.cc](mailto:fae@seeed.cc).

If you want to discuss the product, I recommend the [Seeed Studio Forum](https://forum.seeedstudio.com/) and [Seeed Studio Discord Channel](https://discord.gg/eWkprNDMU7)

The "Discussion channel" in Github here is a relatively new addition and may not receive much attention.

Nonetheless, we hope you find it helpful with the help of others. We will clean it up on a monthly basis.

**Comment by LuzFernandaDominguez** [comment]
Hello!
Could you provide the programming and connection on Raspberry Pi with the hat grove, please?
